src/store/authorization/actions.ts

```diff
@@ -35,20 +39,46 @@ export const actions: ActionTree<AuthorizationStateInterface, RootStateInterface
         return payload.errorCallback(r);
       });
   },
-  [AuthActionEnum.GET_DATA]: ({ commit }, payload) => {
-    HTTP.get(`/users/${payload}`).then((r: AxiosResponse<UserInterface>) => {
+  [AuthActionEnum.GET_DATA]: async ({ commit }, payload) => {
+    await HTTP.get(`/users/${payload}`).then((r: AxiosResponse<UserInterface>) => {
       commit(AuthMutationEnum.SET_USER, r.data);
     });
   },
-  [AuthActionEnum.UPDATE_USER]: ({ commit }, payload) => {
-    HTTP.get(`/users/${payload}`).then((r: AxiosResponse<UserInterface>) => {
-      commit(AuthMutationEnum.SET_USER, r.data);
-    });
 
-    commit(AuthMutationEnum.SET_USER, null);
-  },
   [AuthActionEnum.SIGN_OUT]: ({ commit, dispatch }, payload) => {
     dispatch('localStorageStore/REMOVE', 'user', { root: true });
     commit(AuthMutationEnum.SET_USER, payload);
   },
+  /////////// MY ACTIONS
+  [AuthActionEnum.SET_FORM]: ({ commit }, payload) => {
+    commit(AuthMutationEnum.SET_FORM, payload);
```

Comment:
Why do you need this action? You can call mutation directly from component.

Reply:
I agree. Already changed it.